packages/@glimmer/test-helpers/lib/suites/components.ts

```diff
@@ -163,4 +164,382 @@ export class BasicComponents extends RenderTest {
     this.assertHTML("<div color='red'>left - hello! yield me</div>");
     this.assertStableNodes();
   }
+
+  @test({
```

Comment:
Aren't all of these testing Glimmer Components?

Reply:
I can change these to say `type: 'glimmer'`, but the reason I used `basic` was that _all_ of the other tests in this class were listed as `basic` and the name of the class is `BasicComponents`...

Reply:
I am also confused about the intent of basic components.

Reply:
@wycats or @chancancode should confirm, but "basic" started out as component type that didn't have all the capabilities that a `EmberishGlimmerComponent` or a `EmberishCurlyComponent` had. Was used as quick way to just test the mechanics of the VM. This concept may not matter any more, in that case we should purge the type of the test suite. Likely will simplify things.